### PR TITLE
Candidate for 5.0.0

### PR DESCRIPTION
**Pull request by kushti**
Candidate for 5.0 release. It contains Sigma interpreter 5.0 https://github.com/ScorexFoundation/sigmastate-interpreter/releases/tag/v5.0.0 , new PaiNet testnet settings, and also EIP-39 (monotonic creation height rule) implementation: 
 
* #1865

### CODE REVIEW

**Review by aslesarenko — CHANGES_REQUESTED** [review]

File: src/main/resources/testnet.conf [hunk 1]
```diff
@@ -75,18 +77,23 @@ ergo {
...
+  }
+
   wallet.secretStorage.secretDir = ${ergo.directory}"/wallet/keystore"
 }
 
 scorex {
   network {
-    magicBytes = [2, 0, 0, 3]
-    bindAddress = "0.0.0.0:9021"
+    magicBytes = [2, 0, 2, 3]
```
[the remark this answers is not in the document]
Comment (kushti) [reply src/main/resources/testnet.conf:89]: There is no schema, a new network just need to have new magic bytes. So I do not quite understand what are you asking to document

**Comment by kushti** [comment]
@aslesarenko comments addressed, please make another pass

File: src/main/resources/testnet.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (aslesarenko) [reply src/main/resources/testnet.conf:89]: I think you can put this comment to the code "There is no schema, a new network just need to have new magic bytes."

File: src/test/scala/org/ergoplatform/modifiers/mempool/ErgoTransactionSpec.scala [hunk 2]
```diff
@@ -36,6 +36,52 @@ class ErgoTransactionSpec extends ErgoPropertyTest with ErgoTestConstants {
...
+      if (java.util.Arrays.equals(id, assetId)) assetId -> deltaFn(amount) else assetId -> amount
+    }
+
+    new ErgoBoxCandidate(
+      boxCandidate.value,
+      boxCandidate.ergoTree,
+      boxCandidate.creationHeight,
+      tokens,
+      boxCandidate.additionalRegisters)
+  }
```
[the remark this answers is not in the document]
Comment (aslesarenko) [reply src/test/scala/org/ergoplatform/modifiers/mempool/ErgoTransactionSpec.scala:79]: Well, but the concern still the same.

**Review by aslesarenko — APPROVED** [review]
LGTM, but see unresolved conversations.

**Review by pragmaxim — APPROVED** [review]

File: src/main/resources/testnet.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (kushti) [reply src/main/resources/testnet.conf:89]: The comment is already specifying the goal of the bytes, i.e. distinguishing one network from another. If other properties are not stated, they are not needed.

**Pull request merged**